src/index.html

```diff
@@ -16,17 +16,34 @@
       href="style.css"
     />
   </head>
+
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      class="search__form"
+      data-qa="big"
+    >
+      <input
+        id="big-search-input"
+        class="search__input"
+        type="text"
+        placeholder="Try “Los Angeles“"
+        data-qa="keypress"
+      />
+    </form>
```

Comment:
The `action` attribute in the form tag is set to '#'. This is not a valid URL. If you don't want the form to send data anywhere, you can omit the `action` attribute.